org.opent2t.sample.hub.superpopular/com.hue.hub/js/thingTranslator.js

```diff
@@ -15,43 +15,188 @@ class Translator {
         this._accessToken = accessTokenInfo;
         this._baseUrl = 'https://api.meethue.com/v2/bridges/' + accessTokenInfo.bridgeId + '/' + accessTokenInfo.whitelistId;
         this._devicesPath = '/lights';
+        this._refreshUrl = 'https://api.meethue.com/oauth2/refresh?&grant_type=refresh_token';
```

Comment:
>https://api.meethue.com/oauth2/refresh?&grant_type=refresh_to [](start = 28, length = 61)

what i meant was to take this RHS value from being hardcoded to class or file scoped const (depending on whether its used just in this class or elsewhere. )